Formula/circleci.rb

```diff
@@ -0,0 +1,35 @@
+class Circleci < Formula
```

Comment:
Unrelated file `circleci.rb` needs to be removed.
Also, please squash everything into one commit.